.yamllint

```diff
@@ -0,0 +1,29 @@
+---
+
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+  - '.yamllint'
+
+rules:
+  braces: enable
```

Comment:
a lot of these options are in the default config: https://yamllint.readthedocs.io/en/stable/configuration.html#default-configuration

We should only include non-default rules, otherwise its a hassle to maintain as it looks like you've put them in here for a reason (also makes it hard to review).

Reply:
Fair point. Wasn’t sure if a config file was in addition to, or a complete replacement (so defaults no longer used at all). Will test some more.

Reply:
Ah see in that it extends. Cool. Will remove the defaults.

Reply:
OK I had to add `extends: default` and now it extends :-)